client/resource_group/controller/config.go

```diff
@@ -45,7 +45,9 @@ const (
 	consumptionsReportingThreshold = 100
 	extendedReportingPeriodFactor  = 4
 	// defaultGroupCleanupInterval is the interval to clean up the deleted resource groups in memory.
-	defaultGroupCleanupInterval = 10 * time.Minute
+	defaultGroupCleanupInterval = 5 * time.Minute
+	// defaultSlotStalePeriod is the interval to clean up the stale resource groups in memory.
+	defaultSlotStalePeriod = 5
```

Comment:
What is the unit of this const? It's best to reflect the unit in the name or explain it through comments.

Reply:
Previously it was intended to remove resource group which was not changed more than the const defaultSlotStalePeriod( now is 6 * 5min) , otherwise it was just +1 to reach this const
How much times do you think would be better to clear

Reply:
@CabinfeverB What do you think a better time to remove?

Reply:
I agree this implement, because we don't have a var to save consumption change time. But I prefer `defalutStaleCheckThreshold`, and add some comments.

Reply:
maybe 10min